AppleMusicDiscordRPC/Modals/DiscordRPCObservable.swift

```diff
@@ -121,11 +121,13 @@ class DiscordRPCObservable: ObservableObject {
                 self.artwork.album = album
                 // if no artworks embbeded to the file, it returns nil so RPCStatusView can render no artwork
                 self.artwork.url = artworks?.first?.data ?? nil
+                
+                self.rpc.setPresence(presence)
```

Comment:
I don't think this does anything, since the presence setting happens inside the dataTask closure after we've fetched the artwork from iTunes.